src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Data/Models/Todo/Todo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using TodoTemplate.Shared.Enums;
+
+namespace TodoTemplate.Api.Data.Models.Todo
+{
+    public class Todo
```

Comment:
it's better to not use these general names for our models.
a better name would be `TodoItem`